geometry_msgs/Point point
geometry_msgs/Quaternion rotation
std_msgs/Float64 distance_threshold
std_msgs/Float64 rotation_threshold
std_msgs/String target_frame
std_msgs/String reference_frame
---

---
